server/service/src/item_stats.rs

```diff
@@ -43,22 +42,37 @@ impl ItemStatsServiceTrait for ItemStatsService {}
 pub fn get_item_stats(
     ctx: &ServiceContext,
     store_id: &str,
-    amc_lookback_months: Option<u32>,
+    amc_lookback_months: Option<f64>,
     filter: Option<ItemStatsFilter>,
 ) -> Result<Vec<ItemStats>, RepositoryError> {
     let ItemStatsFilter {
         item_id: item_id_filter,
     } = filter.unwrap_or_default();
 
-    let amc_lookback_months = amc_lookback_months.unwrap_or(DEFAULT_AMC_LOOKBACK_MONTHS);
+    let default_amc = StorePreferenceRowRepository::new(&ctx.connection)
```

Comment:
:nit

`default_amc_lookback_months` makes more sense to me - since the value isn't the 'amc' is the 'number of months'